src/test/java/HW2/RPNTest.java

```diff
@@ -0,0 +1,39 @@
+package HW2;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RPNTest {
+
+    @Test
+    void testBasicEquation() {
```

Comment:
An old convention was to begin every test method with the word test.
That isn't required anymore and should be removed.